addons/web/static/tests/views/fields/ace_editor_field_tests.js

```diff
@@ -259,4 +259,36 @@ QUnit.module("Fields", (hooks) => {
         await click(target, ".o_form_button_save");
         assert.verifySteps(['web_save: [[1],{"foo":"a"}]']);
     });
+
+    QUnit.test("Save and Discard buttons will become invisible after saving", async (assert) => {
+        serverData.models.partner.records.forEach((rec) => {
+            rec.foo = false;
+        });
```

Comment:
you don't need to write on all records here (I'm not even sure why you need to unset the one you'll edit)